static/less/style.less

```diff
@@ -1,4 +1,8 @@
-@import "colors.less";
```

Comment:
We got regressions all over this file. Maybe a merge messed up things.

Reply:
I will see if I can fix this, thought I merged correctly.

Reply:
This should be fixed now